trollmoves/movers.py

```diff
@@ -329,7 +343,13 @@ def open_connection(self):
             ssh_connection.close()
             time.sleep(2)
             LOGGER.debug("Retrying ssh connect ...")
-        raise IOError("Failed to ssh connect after 3 attempts")
+            if retries == 0 and backup_targets:
+                backup_target = backup_targets.pop(0)
+                self.destination = self.destination._replace(netloc=f"{self.destination.username}@{backup_target}")
```

Comment:
I was going to say using private methods might not be a good idea, but apparently this is *public* API. From https://docs.python.org/3/library/collections.html#collections.namedtuple

> To prevent conflicts with field names, the method and attribute names start with an underscore.

Reply:
Yeah I also figured out that.